packages/syntax/src/recordkey.ts

```diff
@@ -0,0 +1,26 @@
+export const ensureValidRecordKey = (rkey: string): void => {
+  if (rkey.length > 512 || rkey.length < 1) {
```

Comment:
i'm pretty tempted to limit this to less than 512

I think we talked about 16 at one point (tho i think that's too low). but maybe 64?

Reply:
I'm not totally against that, but there are a couple things to discuss. I opened an issue in the specs repo: https://github.com/bluesky-social/atproto-website/issues/250

Reply:
cool cool - that doesn't have to hold this up 👍